src/views/pacbio/PacbioLibraryIndex.vue

```diff
@@ -57,125 +57,116 @@
   </DataFetcher>
 </template>
 
-<script>
-import PrinterModal from '@/components/PrinterModal'
-import FilterCard from '@/components/FilterCard'
-import DataFetcher from '@/components/DataFetcher'
-import { mapActions, mapGetters } from 'vuex'
-import { getCurrentDate } from '@/lib/DateHelpers'
-import useQueryParams from '@/lib/QueryParamsHelper'
+<script setup>
+import PrinterModal from '@/components/PrinterModal.vue'
+import FilterCard from '@/components/FilterCard.vue'
+import DataFetcher from '@/components/DataFetcher.vue'
+import { getCurrentDate } from '@/lib/DateHelpers.js'
+import useQueryParams from '@/lib/QueryParamsHelper.js'
```

Comment:
Is it worth moving useQueryParams form src/lib to src/composables not that composables has a designated folder?